### PR TITLE
Add Avro schema for logging

### PR DESCRIPTION
**Pull request by Peiling-Ding**

### CODE REVIEW

File: logging/schema.json [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      "type": "map",
+      "values": "string"
+    },
+    {
+      "name": "response_context",
+      "type": {
+        "type": "record",
+        "fields": [
+          {
+            "name": "winner_bid",
```
Comment (shunj-nb) [inline logging/schema.json:50]: winning_bid should be long type?

File: logging/schema.json [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+      "values": "string"
+    },
+    {
+      "name": "bidder_context_list",
+      "type": {
+        "type": "array",
+        "items": {
+          "name": "bidder_context",
+          "type": "record",
+          "fields": [
```
Comment (shunj-nb) [inline logging/schema.json:76]: what about adding a boolean value like isWin here to indicate the winner

**Comment by StarWindMoonCloud** [comment]
are we merging request and response
is it num of bidders records for each ad slot (opportunity)

**Review by archzhou — APPROVED** [review]
LGTM. Just confirm if we need also to log the creative ids and campaign ids.

**Review by StarWindMoonCloud — APPROVED** [review]

File: logging/schema.json [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+            {
+              "name": "request_context",
+              "type": "map",
+              "values": "string"
+            },
+            {
+              "name": "response_code",
+              "type": "string"
+            },
+            {
```
[the remark this answers is not in the document]
Comment (Peiling-Ding) [reply logging/schema.json]: No, we can't remove a field from a schema, it is not backward compatible. But we can make it nullable.

[the remark this answers is not in the document]
Comment (Peiling-Ding) [reply logging/schema.json]: > LGTM. Just confirm if we need also to log the creative ids and campaign ids.

Added

**Pull request merged**